geoh5py/objects/points.py

```diff
@@ -32,12 +32,46 @@ class Points(ObjectBase):
     """
 
     __TYPE_UID = uuid.UUID("{202C5DB1-A56D-4004-9CAD-BAAFD8899406}")
+    __VERTICES_DTYPE = np.dtype([("x", "<f8"), ("y", "<f8"), ("z", "<f8")])
 
-    def __init__(self, object_type: ObjectType, name="Points", **kwargs):
-        self._vertices: np.ndarray | None = None
+    def __init__(
```

Comment:
The extent function can be set in points for all the inherited objects

Reply:
Yep, done.